tensorflow/tools/ci_build/pi/build_raspberry_pi.sh

```diff
@@ -118,8 +118,6 @@ SUB='s/tensorflow-([^-]+)-([^-]+)-.*/tensorflow-\1-\2-none-'${WHEEL_ARCH}'.whl/;
 NEW_FN=$(echo "${OLD_FN}" | perl -ne "${SUB}")
 mv "${OUTDIR}/${OLD_FN}" "${OUTDIR}/${NEW_FN}"
 cp bazel-bin/tensorflow/tools/benchmark/benchmark_model "${OUTDIR}"
-cp bazel-bin/tensorflow/libtensorflow.so "${OUTDIR}"
```

Comment:
These two lines are actually added by https://github.com/tensorflow/tensorflow/commit/0d697e5fc4c05c699eea0764364104ea500ccc68#diff-00a21b168974d2d842d84a57987129f4
@jessebenson added these for #17850

Reply:
Thanks for spotting that. It looks like a merge wiped out the added targets:
https://github.com/tensorflow/tensorflow/commit/80e7bedb36ddacdbacd70b8bcab26eb911081425#diff-00a21b168974d2d842d84a57987129f4

I'll update this PR to add those targets back in, rather than removing the copies.